.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
this should be just empty line and not the underscore, right?

Reply:
No, hubot puts a file in that folder and that needs to be ignored by git. This gitignore is automatically generated by husky